src/systemctl/systemctl.c

```diff
@@ -3548,6 +3548,22 @@ static int load_kexec_kernel(void) {
         return 0;
 }
 
+static int kill_user_manager(void) {
+        int r;
+        char *pid_env;
+        pid_t pid;
+
+        pid_env = getenv("MANAGERPID");
+        if (!pid_env || !*pid_env)
+                return -EINVAL;
+        r = parse_pid(pid_env, &pid);
+        if (r < 0)
+                return r;
```

Comment:
I think we should log something in this case.

Reply:
Done.